ql/src/java/org/apache/hadoop/hive/ql/optimizer/SortedDynPartitionOptimizer.java

```diff
@@ -276,8 +283,7 @@ public Object process(Node nd, Stack<Node> stack, NodeProcessorCtx procCtx,
 
       // Create ReduceSink operator
       ReduceSinkOperator rsOp = getReduceSinkOp(partitionPositions, sortPositions, sortOrder, sortNullOrder,
-                                                  allRSCols, bucketColumns, numBuckets,
-                                                 fsParent, fsOp.getConf().getWriteType());
```

Comment:
This line shows up a conflict in some of my merges - is this just a white-space change?

Reply:
Yes, that's correct, it's only white-space/merge into single row.